src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &--1 :first-child,
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(-n + 5) {
```

Comment:
The selectors `&--1 :first-child`, `&--2 :nth-child(-n + 2)`, `&--3 :nth-child(-n + 3)`, `&--4 :nth-child(-n + 4)`, `&--5 :nth-child(-n + 5)` are not quite correct. They should be `&--1 &__star:first-child`, `&--2 &__star:nth-child(-n + 2)`, `&--3 &__star:nth-child(-n + 3)`, `&--4 &__star:nth-child(-n + 4)`, `&--5 &__star:nth-child(-n + 5)`. This is because you want to select the `__star` element that is a child of the `stars` element with the modifier class `--1`, `--2`, `--3`, `--4`, or `--5`.